### PR TITLE
New Presence, Pilli Oyun (pillioyun.com) 

### PR DESCRIPTION
**Pull request by KeremWho**
**Work Preview:** https://streamable.com/v4dk2t

### CODE REVIEW

File: websites/P/Pilli Oyun/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+  "service": "Pilli Oyun",
+  "author": {
+    "name": "KeremWho",
+    "id": "179997163294556161"
+  },
+  "category": "other",
+  "logo": "https://pillioyun.com/wp-content/uploads/2021/03/Favicon.png",
+  "thumbnail": "https://pillioyun.com/premid-assets/screenshot.png",
```
Comment (SlowLife1661) [inline websites/P/Pilli Oyun/dist/metadata.json:9]: Upload to imgur

**Review by SlowLife1661 — CHANGES_REQUESTED** [review]

**Comment by SlowLife1661** [comment]
Why not make a presenceData variable?

File: websites/P/Pilli Oyun/dist/metadata.json (hunk 1 above)
Comment (KeremWho) [reply websites/P/Pilli Oyun/dist/metadata.json]: I did.

**Comment by KeremWho** [comment]
> Why not make a presenceData variable, and then use setActivity() once?

I took PlayerBros' presence as an example.

**Review by SlowLife1661 — CHANGES_REQUESTED** [review]

**Comment by Bas950** [comment]
The website/domain isnt old enough.

**Pull request closed without merging**